assume/common/utils.py

```diff
@@ -66,7 +66,7 @@ def plot_orderbook(orderbook: Orderbook, results: list[dict]):
     from matplotlib.lines import Line2D
 
     bids = defaultdict(list)
-    orderbook = sorted(orderbook, key=itemgetter("node_id"))
+    orderbook.sort(key=itemgetter("node_id"))
```

Comment:
I find always better to have x = x+1 formulations rather then inplace formulations. Somehow its easier to read those and understand. is there a particular reason for such formulation?